demo/main.gd

```diff
@@ -10,7 +10,7 @@ func _ready():
 		var vp: Viewport = get_viewport()
 		vp.use_xr = true
 		
-	scene_capture = OpenXRFbSceneCaptureExtensionWrapper.get_singleton()
+	scene_capture = Engine.get_singleton("OpenXRFbSceneCaptureExtensionWrapper")
```

Comment:
You shouldn't need to do the `Engine.get_singleton()` bit, in GDScript all singletons should be available like globals, ie `OpenXRFbSceneCaptureExtensionWrapper.request_scene_capture()` should work to call that method. Although, I would also have expected `Engine.get_singleton()` to work too...